### PR TITLE
Add influx storage and check rich guy

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

**Comment by halink0803** [comment]
[Implement users component](https://trello.com/c/Ras0ZhqQ/253-implement-users-component)

File: users/storage/influx.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	inf.sugar.Debugw("result from query", "result", res)
+
+	if err != nil {
+		return false, err
+	}
+	var userTradeAmount float64
+	if len(res[0].Series) > 0 {
+		userTradeAmount = (res[0].Series[0].Values[0][1]).(float64)
+	}
+	return userTradeAmount >= dailyLimit, nil
```
Comment (favadi) [inline users/storage/influx.go]: please add a debug log like: 

- `il.sugar.Debugw("user is rich", "last_24h_amount",  "address", address, userTradeAmount, "daily_limit", dailyLimit)`
- `il.sugar.Debugw("user is not rich", "last_24h_amount",  "address", address, userTradeAmount, "daily_limit", dailyLimit)`

File: users/http/server.go [hunk 2]
```diff
@@ -61,13 +67,22 @@ func (s *Server) getTransactionLimit(c *gin.Context) {
 	// maximum of ETH in wei
 	uc := common.NewUserCap(kyced)
 	txLimit := blockchain.EthToWei(uc.TxLimit / rate)
+	rich, err := s.influxStorage.IsExceedDailyLimit(address, uc.DailyLimit)
```
[the remark this answers is not in the document]
Comment (halink0803) [reply users/http/server.go]: if there is no record with matching address then it means the user have not trade anytime, so the `rich` would be `false` and the `err` is `nil`.

[the remark this answers is not in the document]
Comment (favadi) [reply users/http/server.go]: So you are confirmed that when there is no record it returns "false, nil". Maybe better to prove with a test case.

**Pull request merged**